cli/core/status.go

```diff
@@ -144,11 +147,20 @@ func GetStatus(ctx context.Context, eigenpodAddress string, eth *ethclient.Clien
 	proofSubmitter, err := eigenPod.ProofSubmitter(nil)
 	PanicOnError("failed to get eigenpod proof submitter", err)
 
-	currentOwnerShares, err := eigenPodManager.PodOwnerShares(nil, eigenPodOwner)
-	// currentOwnerShares = big.NewInt(0)
-	PanicOnError("failed to load pod owner shares", err)
-	currentOwnerSharesETH := IweiToEther(currentOwnerShares)
-	currentOwnerSharesWei := currentOwnerShares
+	delegationManagerAddress, err := eigenPodManager.DelegationManager(nil)
+	PanicOnError("failed to read delegationManager", err)
+
+	delegationManager, err := DelegationManager.NewDelegationManager(delegationManagerAddress, eth)
+	PanicOnError("failed to reach delegationManager", err)
+
+	_, shares, err := delegationManager.GetDepositedShares(nil, eigenPodOwner)
+	PanicOnError("failed to load owner shares", err)
```

Comment:
The return value here is an array of shares / array of strategies. If you're using this method you only want to sum the shares that correspond to the beacon chain ETH strategy.

That said, I think you want `getWithdrawableShares`, which accepts a strategy as input and will return (withdrawableShares, depositShares)